api/android/api/src/main/jni/nnstreamer-native-customfilter.c

```diff
@@ -137,8 +152,10 @@ nns_customfilter_invoke (const GstTensorFilterProperties * prop, void **private_
     (*env)->DeleteLocalRef (env, obj_out_info);
   (*env)->DeleteLocalRef (env, cls_custom);
 
-  g_free (in_data);
-  g_free (out_data);
+  if (in_data)
```

Comment:
No need of NULL check before g_free
https://developer.gnome.org/glib/stable/glib-Memory-Allocation.html#g-free

Reply:
aah, this is duplicate.